src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery[] lotteries = {new Lottery(), new Lottery(), new Lottery()};
```

Comment:
you need only one lottery here
```suggestion
        Lottery lottery = ...;

Reply:
previous mentor declined my solution with:
Lottery lottery1 = new Lottery();
        Lottery lottery2 = new Lottery();
        Lottery lottery3 = new Lottery();

System.out.println(lottery1.getRandomBall());
        System.out.println(lottery2.getRandomBall());
        System.out.println(lottery3.getRandomBall());